Documentation/RuleDocumentation.md

```diff
@@ -0,0 +1,478 @@
+<!-- This file is autogenerated with `generate-pipeline` utility of
+`swift-format`. DO NOT EDIT MANUALLY! -->
+
+# `swift-format` Lint and Format Rules
+
+Use the rules below in the `rules` block of your `.swift-format`
+configuration file, as described in
+[Configuration](Documentation/Configuration.md). All of these rules can be
+applied in the linter, but only some of them can format your source code
+automatically.
+
```

Comment:
Could you generate an unordered list here that would act as a table of contents, linking to each of the headers below?